<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget{
background-color: rgb(230,230,230);
}

QGroupBox{
color:  rgb(80, 0, 19);  /* Text color */
font-weight: bold;  /* Bold text */
}

QLabel{
color:  rgb(80, 0, 19);;  /* Text color */
}

QPushButton {
	background-color: rgb(122,0,28);  /* Background color */
	border: 1px solid rgb(0, 0, 0);
    color: rgb(255, 255, 255);  /* Text color */
    border-radius: 5px;  /* Rounded edges */
	padding: 2px;
}

QPushButton:hover {
    background-color: rgb(142, 0, 38);  /* Background color when hovered */
	color: rgb(245, 245, 245);
    border-color:rgb(40, 40, 40);  /* Border color when hovered */
}

QPushButton:pressed {
   color:  rgb(80, 0, 19);  /* Background color when pressed */
	border-color: rgb(0, 0, 0);
}
QPushButton:clicked {
   background-color: rgb(80,0,19);  /* Background color when pressed */
	border-color:  rgb(0,0, 0);
}

QPushButton:checked {
    background-color: rgb(255,255,255);
	color:  rgb(80, 0, 19);
	border: 2px solid rgb(122, 0, 28);
	font-weight: bold;  /* Bold text */
}

QPushButton:disabled {
    background-color: rgb(180,180, 180);
	border: 1px solid rgb(0, 0,0);
}

QDoubleSpinBox:enabled{
background-color: rgb(255,255,255);
color:  rgb(80, 0, 19);  /* Text color */
border: 1px solid  rgb(122, 0, 28);  /* Border color */
border-radius: 5px;  /* Rounded edges */
padding : 2px;
}
QDoubleSpinBox:disabled{
color: rgb(80, 0, 19);  /* Text color */
border: 1px solid rgb(255,255,255);  /* Border color */
border-radius: 5px;  /* Rounded edges */
padding : 2px;
}

QSpinBox:enabled{
background-color: rgb(255,255,255);
color:  rgb(80, 0, 19);  /* Text color */
border: 1px solid  rgb(122, 0, 28);  /* Border color */
border-radius: 5px;  /* Rounded edges */
padding : 2px;

}
QSpinBox:disabled{
color: rgb(80, 0, 19);  /* Text color */
border: 1px solid rgb(255,255,255);  /* Border color */
border-radius: 5px;  /* Rounded edges */
padding : 2px;
}

QPlainTextEdit{
background-color: rgb(255,255,255);
border-radius: 5px;  /* Rounded edges */
}

QLineEdit:endabled{
background-color: rgb(255,255,255);
color:  rgb(80, 0, 19);  /* Text color */
border: 1px solid rgb(0,0,0);  /* Border color */
border-radius: 5px;  /* Rounded edges */
padding : 2px;
}

QLineEdit:disabled{
background-color: rgb(230,230,230);
color:  rgb(80, 0, 19);  /* Text color */
border: 1px solid rgb(0,0,0);  /* Border color */
border-radius: 5px;  /* Rounded edges */
padding : 2px;
}

QComboBox{
background-color: rgb(255,255,255);
color:  rgb(80, 0, 19);  /* Text color */
border: 1px solid rgb(0,0,0);  /* Border color */
border-radius: 5px;  /* Rounded edges */
padding : 2px;
}

QTabBar::tab {
    color:  rgb(80, 0, 19); 
    border: 1px solid rgb(122,0,28);
    padding: 5px;
    border-top-left-radius: 5px;
    border-top-right-radius: 5px;
	min-width:80px;
}

QTabBar::tab:selected {
    color:  rgb(80, 0, 19); 
    background-color: rgb(255, 255, 255); ;
    font-weight: bold;
	border: 2px solid rgb(122, 0, 28);
}

QTabBar::tab:hover {
    color:  rgb(122, 0, 28); /* Slightly lighter orange */
	background-color: rgb(245, 245, 245); ;
}
</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="1" column="0">
    <widget class="QPushButton" name="add_point_button">
     <property name="text">
      <string>Add Point</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QListWidget" name="points_listWidget"/>
   </item>
   <item row="2" column="0">
    <widget class="QPushButton" name="calc_transform_button">
     <property name="text">
      <string>Calculate Transform Matrix</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QPushButton" name="save_default_button">
     <property name="text">
      <string>Save Current Matrix to Default Settings</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
